### ISSUE TITLE
Issue with storms at end of years when excluding years

### ISSUE THREAD

**Issue by blfitzgerald** [HydroclimateExtremesGroup/RainyDay#3]
Ashar, 
I was running the code on CHTC separately by year and was getting slightly different results because storms that were on the end of the year weren't able to cross over. This isn't really a bug but Dan wanted me to document it. This issue probably occurs when running the original code on just a year or when excluding years.  
Ben

**Comment by danielbwright** [comment]
I agree that this is worth addressing. Basically, when "INCLUDEYEARS" is set to something other than "all", let's make sure the first T timesteps of the next year are included in the input file list. So for example, if you do "INCLUDEYEARS": 2014 and duration is 48 hours, let's include the first 48 hours of 2015. I am envisioning this as a step towards a potential way of parallelizing RainyDay, in which we run the code simultaneously for individual years or subsets of years, and then combine the results to get the storm catalog (this is what Ben is attempting to do; would help in applications such as Jack's and Daniel L's, etc.)

**Comment by Ashar25** [comment]
Sorry, this issue is old but I didn't get any mail about this issue, I am not sure why. We can discuss about this issue. Right now, I am not sure how to handle this.

**Comment by blfitzgerald** [comment]
I didn't hear any update about it @danielbwright

**Comment by Ashar25** [comment]
I couldn’t understand this issue fully. It will be good if we discuss it and then I can resolve it. Doesn’t look like a major issue.
